client/my-sites/earn/index.ts

```diff
@@ -12,7 +12,7 @@ import {
 const earnPath = ! isJetpackCloud() ? '/earn' : '/monetize';
 
 export default function () {
-	page( '/earn', siteSelection, sites, makeLayout, clientRender );
+	page( earnPath, siteSelection, sites, makeLayout, clientRender );
```

Comment:
Coming from https://github.com/Automattic/wp-calypso/pull/89390#pullrequestreview-1991418666
Noticed it here now 🙏